### PR TITLE
FAQ EFGS - DK until 01.04.2022

### PR DESCRIPTION
**Pull request by dsarkar**
Internal Tracking ID: [EXPOSUREAPP-12301](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12301)
Internal Tracking ID: [EXPOSUREBACK-2145](https://jira-ibs.wbs.net.sap/browse/EXPOSUREBACK-2145)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -1415,8 +1415,8 @@
...
+                                    "(Status: 21.05.2022)",
```
Comment (Ein-Tim) [inline src/data/faq.json:1418]:
```suggestion
                                    "(Status: 02.05.2022)",
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Typo

**Review by svengabr — APPROVED** [review]

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json:1418]: hey @Ein-Tim Yep, thanks, typo!

**Comment by dsarkar** [comment]
@Ein-Tim thanks!

**Pull request merged**